### PR TITLE
Display last update timestamp in local time

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This creates a method to localize the `LastUpdate` timestamp as it is currently stored in UTC.

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

The 1st envelope to appear in the table shown in the image was created at 6:05 PM ET.
https://tinyurl.com/2yphxrxz

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/templates/partials/transaction/transaction_list.html [hunk 1]
```diff
@@ -109,10 +109,8 @@
         {{ $created := .Created.Format "January 2, 2006 15:04:05" }}
         <td>{{ $created }}</td>
         {{ if .LastUpdate }}
-        {{ $lastUpdate := .LastUpdate.Format "January 2, 2006 15:04:05" }}
+        {{ $lastUpdate := .LocalizeLastUpdate.Format "January 2, 2006 15:04:05" }}
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/transaction/transaction_list.html:112]: In my timezone, I would see `in 4 hours` in the `last update` column after sending an envelope. Localizing the time should resolve this issue.

**Review by bbengfort — COMMENTED** [review]
Unfortunately I don't think this method will work unless we can get the timezone information from the user (e.g. maybe we set it as a cookie somehow so that the server knows what the user's timezone is?). 

The central problem is that Javascript is treating the lat update timestamp as the same timezone as the user, so when it does time math, it is incorrect (unless the user is in UTC time). If we added the timezone to the timestamp, would Javascript handle it correctly?

File: pkg/web/templates/partials/transaction/transaction_list.html (hunk 1 above)
Comment (daniellemaxwell) [reply pkg/web/templates/partials/transaction/transaction_list.html]: Using the offset appears to work. I took the following screenshot a few minutes after sending a new envelope: https://tinyurl.com/29a7kmh3

**Review by bbengfort — APPROVED** [review]
Glad that fixed it!

**Pull request merged**